### ISSUE TITLE
Is this project functional?

### ISSUE THREAD

**Issue by stoefln** [nanoscopic/ios_video_stream#2]
Judging by the amount of code, documentation, PRs and tickets I'd say this is WIP, right?
If not: I would be really interested in streaming content from ios to macOs. The readme.md is just not enough for me to set it  up...

**Comment by nanoscopic** [comment]
It works, or at least it worked last time I tried it, which was, admittedly, over 4 months ago.

In theory, if you follow the instructions of the stf_ios_support repo, you can have video streaming working with STF.

In reality, it won't work because STF has changed since last I was maintaining stf_ios_support, so it won't work against updated STF code. ( you could use old STF code and it would work... )

All that said, there is essentially no point to attempting to use this repo, because:

1. I've updated / rewritten / ported all of the code in this repo directly into the new ios_remote_provider repo.
2. stf_ios_support itself is essentially deprecated. I moved on to using ios_remote_provider instead.
3. ios_remote_provider works with the new providing system I wrote, ControlFloor. ( see controlfloor repo )

The thing you probably want to look at is ios_video_app.
In theory, ios_video_app supports working with this older project ( ios_video_stream ). I did write ios_video_app initially to support doing that. Since then though I've shifted to only making it work with the newer version of this code within ios_remote_provider.

TLDR: Use ControlFloor, ios_remote_provider, and ios_video_app instead of this repo.
